src/clean.js

```diff
@@ -111,6 +111,14 @@ function clean(node, newObj) {
   if (node.kind === "useitem") {
     newObj.name = newObj.name.replace(/^\\/, "");
   }
+  if (
+    node.kind === "string" &&
+    node.value &&
+    (node.value[0] === "'" || node.value[0] === '"') &&
+    (node.value[node.length - 1] !== "'" || node.value[node.length - 1] !== '"')
```

Comment:
isn't the expression `(node.value[node.length - 1] !== "'" || node.value[node.length - 1] !== '"')` always `true` ?

Reply:
Yep, I read this the same way. Also, it might be nicer to use `getLast` instead of `arr[arr.length - 1]`

Reply:
@evilebottnawi What do you think about this one?

Reply:
@czosel simplify and remove unnecessary check, also add more tests

Reply:
:+1: